rosidl_parser/rosidl_parser/definition.py

```diff
@@ -551,6 +551,10 @@ def __init__(self, type_, goal_request, result_response, feedback):
                 Member(feedback.structure.type, 'feedback')]
         ))
 
+        self.includes = [
+            Include('builtin_interfaces/msg/Time.idl')
+        ]
+
```

Comment:
Includes are not specific to an `Action` but are available through the `IdlContent` instance. Therefore I don't think this should be added to the `Action` class.

Reply:
Since it is the `Action` that's adding the field, I thought it was cleaner to keep it as an implementation detail.